db/migrate/20161103101916_create_applications.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :applications do |t|
       t.string :motivation
       t.references :user, index: true, foreign_key: true
-      t.references :workshop, index: true, foreign_key: true
+      t.references :event, index: true, foreign_key: true
```

Comment:
@ekrebs5 I think the reason it both worked for you is that Ruby is so smart, it doesn't repeat migrations it already applied. Because the filename didn't change, this change was never tested when you just migrated your database. To really check this, you'd have to manually reverse all applied migrations until then, an migrate everything again. I am pretty sure the renaming would fail with this change, because there would be no more ":workshop"-column.